bioinformatics_utils.py

```diff
@@ -0,0 +1,137 @@
+from typing import Union, List, Dict, Tuple
+from bioinformatics_toolkit import dna_rna_tools
+from bioinformatics_toolkit import filter_fastq
+from bioinformatics_toolkit import protein_tool
+
+ALPHABET = {'a', 't', 'u', 'g', 'c', 'A', 'T', 'U', 'G', 'C'}
+
+def run_dna_rna_tools(*args: Union[str, List[str]], ALPHABET: set = ALPHABET) -> Union[str, List[str]]:
+    """
+    Perform the specified operation on the given DNA/RNA sequences.
+    
+    This function performs the following operations: transcribtion, reversation, complementation, 
+    reversal complementation.
+    
+    Parameters:
+        *args (Union[str, List[str]]): DNA/RNA sequence(s).
+        ALPHABET (set): Default set of valid nucleotide characters.
+    
+    Returns:
+        Union[str, List[str]]: The result of the specified operation on the sequence(s).
+            If multiple sequences are provided, a list of results is returned.
+
+    Examples:
+        # Transcribing a single DNA sequence
+        run_dna_rna_tools("ATGCTA", "transcribe")
+        # Result: "AUGCUA"
+
+        # Transcribing multiple DNA sequences
+        run_dna_rna_tools("ATGCTA", "TTAGCG", "transcribe")
+        # Result: ["AUGCUA", "UUAGCG"]
+    """
+    operations = {
+        'transcribe': transcribe,
+        'reverse': reverse,
+        'complement': complement,
+        'reverse_complement': reverse_complement
+    }

```

Comment:
Смотри, (тут и к следующим двум функциям то же самое применимо) ты тут задаешь функции, но ты их не импортировал. Ты импортировал скрипт с ними

У тебя есть два варианта: либо изменить импорты типа

`from bioinformatics_toolkit.dna_rna_tools import transcribe, reverse, reverse_complement, complement`

и тогда такой словарь будет норм работать, либо с прежними импортами сделать словарь типа такого:
```suggestion
    operations = {
        'transcribe': dna_rna_tools.transcribe,
        'reverse': dna_rna_tools.reverse,
        'complement': dna_rna_tools.complement,
        'reverse_complement': dna_rna_tools.reverse_complement
    }

```

Этот комментарий распространяется на все импорты и все функции из тех скриптов, которые есть в этом коде. Надо будет что-то поменять: либо импорт, либо вызов функций.